components/Buttons/src/MDCButton.m

```diff
@@ -262,6 +270,11 @@ - (void)commonMDCButtonInit {
   shadowLayer.shadowColor = [UIColor blackColor].CGColor;
   shadowLayer.elevation = [self elevationForState:self.state];
 
+  if (!_shadowColors) {
```

Comment:
You don't need this if statement.

Reply:
If `_shadowColors` was created in `-initWithCoder:`, we shouldn't overwrite it.

Reply:
Nevermind, I just realized that commonInit is called before the decoding takes place.